lib/resource-providers/vpc.ts

```diff
@@ -29,9 +35,38 @@ export class VpcProvider implements ResourceProvider<ec2.IVpc> {
             // It will automatically divide the provided VPC CIDR range, and create public and private subnets per Availability Zone.
             // Network routing for the public subnets will be configured to allow outbound access directly via an Internet Gateway.
             // Network routing for the private subnets will be configured to allow outbound access via a set of resilient NAT Gateways (one per AZ).
+            // Creates Secondary CIDR and Secondary subnets if passed.
             vpc = new ec2.Vpc(context.scope, id + "-vpc");
+            const secondarySubnets: Array<PrivateSubnet> = [];
+            let tempSecondarSubnet;
+            if (this.secondaryCidr) {
+                new ec2.CfnVPCCidrBlock(context.scope, id + "-secondaryCidr", {
```

Comment:
this is redundant structure. I gave you the code only to assign this to a variable.

Reply:
My bad. Such a long day.